### PR TITLE
[ARO] add 7.1 jobs

### PR DESCRIPTION
**Pull request by mjudeikis**
Add minor release testing jobs

/cc @jim-minter @ehashman

### CODE REVIEW

File: ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-release-v7-presubmits.yaml [hunk 1]
```diff
@@ -237,6 +237,24 @@ presubmits:
         resources: {}
       serviceAccountName: ci-operator
     trigger: (?m)^/test( | .* )e2e-scaleupdown-v7.0,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - release-v7
+    context: e2e-scaleupdown-v7.1
+    decorate: true
+    name: pull-ci-azure-release-v7-e2e-scaleupdown-v7.1
```
Comment (mjudeikis) [inline ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-release-v7-presubmits.yaml]: Do we need this on? O_o

Comment (ehashman) [reply ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-release-v7-presubmits.yaml]: Probably doesn't hurt? Do we want to also add a 7.1 upgrade case?

Comment (mjudeikis) [reply ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-release-v7-presubmits.yaml]: we already have one `pull-ci-azure-release-v7-upgrade-v7.0`

**Comment by Makdaam** [comment]
/lgtm
with the caveat that the scaleupdown test on release-v7 branch might be unnecessary unless we plan on releasing v7.2

**Pull request merged**